azure-pipelines.yml

```diff
@@ -79,3 +84,17 @@ stages:
       submodules: recursive
     - template: templates/azure-pipelines/entrypoint/windows/powershell/continuous-build.yml@PSModulePublisher
     - template: templates/azure-pipelines/entrypoint/windows/powershell/publish.yml@PSModulePublisher
+- stage: release
+  displayName: Release
+  dependsOn: publish
+  jobs:
+  - job: linux_container
+    displayName: '[Linux] [Container] PowerShell Core'
+    pool:
+      vmImage: ubuntu-18.04
+    container: theohbrothers/docker-powershell:7.1.3-ubuntu-18.04-20210316-git
+    steps:
+    - checkout: self
+      submodules: recursive
+    - template: templates/azure-pipelines/entrypoint/generate.yml@PSRepositoryReleaseManager
+    - template: templates/azure-pipelines/entrypoint/release.yml@PSRepositoryReleaseManager
```

Comment:
@joeltimothyoh ci templating is not a good workflow, as most ci providers have realized, which was what i was saying in https://github.com/theohbrothers/ItemLinkManagement/pull/6#issuecomment-890602682